plugins/tiddlywiki/tour/globals.tid

```diff
@@ -36,6 +36,31 @@ text: longer description of the tour
 <$transclude $tiddler={{$:/state/tour/step}} $field="enter-actions"/>
 \end
 
+\procedure tour-chooser(filter:"[all[shadows+tiddlers]tag[$:/tags/Tour]]")
```

Comment:
IMO `\whitespace trim` for every procedure or macros that uses indentation is missing. Did not check any other code. Just this one sprang into the eye

Reply:
Ah. Good to know. I have missed that. Did we document that one?

Reply:
I couldn't find any mention in the docs so made an update in bd99cf3385ef876cfeae8863997948821c7e7c13